### PR TITLE
docs(ReactJS.md): Update Analytics part

### PR DESCRIPTION
**Pull request by Kelv1nYu**
Update Analytics part

### CODE REVIEW

File: FrontEnd/ReactJS.md [hunk 1]
```diff
@@ -181,12 +182,14 @@ Please read the sdk documentation, it is also an [implementaion reference](https
...
-          $title: response.data.title,
           $state: response.data.state,
           $city: response.data.city,
+          $region: reponse.data.region,
           $cell_phone: response.data.cell_phone,
           $company: response.data.company,
-          $device: deviceType()
+          $device: deviceType(),
+          $OS: OSType(),
+          $last_seen: reponse.data.last_seen
```
Comment (ryohang) [inline FrontEnd/ReactJS.md:192]: remove

File: FrontEnd/ReactJS.md [hunk 2]
```diff
@@ -217,6 +221,25 @@ Mixpanel.track("Download File", {"name": filename}});
...
+Mixpanel.track("LogIn Button Clicked");
+
+//good
+Mixpanel.track("LogOut", {"date": date}); // date format MM-DD-YYYY HH:mm:ss AM/PM
+
+//bad
+Mixpanel.track("LogOut Button Clicked");
+
+//good
+Mixpanel.track("pageView", {"pagePath": path, "date": date}); // date format MM-DD-YYYY HH:mm:ss AM/PM
```
Comment (ryohang) [inline FrontEnd/ReactJS.md:238]: we need to change every page.

File: FrontEnd/ReactJS.md [hunk 3]
```diff
@@ -227,13 +250,25 @@ React.useEffect(()=>{
    
      Mixpanel.time_event('Page Stay');
      return () => {
-        Mixpanel.track("Page Stay",{"sponsor": company});
```
Comment (ryohang) [inline FrontEnd/ReactJS.md:230]: we need that in each page.

**Review by ryohang — CHANGES_REQUESTED** [review]

**Comment by Kelv1nYu** [comment]
Changes completed.

**Review by ryohang — APPROVED** [review]

**Pull request merged**